### PR TITLE
Add an issue template

### PR DESCRIPTION
**Pull request by mrackwitz**
This is based on our issue templates we use on our Client SDK repos. It frames the most common question and gives a list of relevant involved variables and versions which provide us helpful data points, so that we can avoid unnecessary back and forth to investigate the scope of problems in case of bugs.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+> We LOVE to help with any issues or bugs you have!
+>
+> **Questions**: If you have questions about how to use Realm, ask on 
+> [StackOverflow](http://stackoverflow.com/questions/ask?tags=realm).
+> We monitor the `realm` tag.
```
Comment (teotwaki) [inline .github/ISSUE_TEMPLATE.md]: This part and above is irrelevant for an internal repo.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+What did you expected to happen?
+
+## Actual Results
+
+What did happened instead?  
+e.g. the stack trace of a crash
+
+## Steps to Reproduce
+
+What are steps we can follow to reproduce this issue?
```
Comment (teotwaki) [inline .github/ISSUE_TEMPLATE.md]: Typo: `the steps`

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Code Sample
+
+Provide a code sample or test case that highlights the issue.
+If relevant, include your model definitions.
+For larger code samples, links to external gists/repositories are preferred.
+Alternatively share confidentially via mail to help@realm.io.
```
Comment (teotwaki) [inline .github/ISSUE_TEMPLATE.md]: This line is irrelevant

**Comment by teotwaki** [comment]
Nevermind, I thought this was on an internal repo. 👍

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+## Actual Results
+
+What did happened instead?  
+e.g. the stack trace of a crash
+
+## Steps to Reproduce
+
+What are steps we can follow to reproduce this issue?
+
+## Code Sample
```
Comment (bmunkholm) [inline .github/ISSUE_TEMPLATE.md]: We could give tips to how to get logs etc - or a link to such a description

Comment (teotwaki) [inline .github/ISSUE_TEMPLATE.md]: PRs welcome ;)

**Pull request merged**